src/test/java/seedu/address/testutil/stubs/ModelStub.java

```diff
@@ -117,4 +117,14 @@ public RoomList getModifiableRoomList() {
     public void addTaskToRoom(Task task, Room room) {
         throw new AssertionError("This method should not be called.");
     }
+
+    @Override
+    public boolean containsRoom(Room room) {
```

Comment:
Also, perhaps we can rename these methods to `hasRoom(...)` and `setRoom(...)` for consistency with `Patient`?

Reply:
i initially used ``hasRoom()`` and ``setRoom()``. i forgot why i didnt use ``hasRoom()`` but i didnt use ``setRoom()`` because roomList class already has setRoom(), i was thinking it will be quite confusing to have 2 methods named so closely to each other but does totally different things